profitbricks-rest/src/main/java/org/apache/jclouds/profitbricks/rest/binder/BaseProfitBricksRequestBinder.java

```diff
@@ -16,20 +16,18 @@
  */
 package org.apache.jclouds.profitbricks.rest.binder;
 
-import static com.google.common.base.Preconditions.checkNotNull;
-
 import java.util.Map;
-
 import org.jclouds.http.HttpRequest;
 import org.jclouds.rest.MapBinder;
-
 import com.google.common.base.Strings;
-
 import org.jclouds.io.MutableContentMetadata;
 import org.jclouds.io.payloads.BaseMutableContentMetadata;
+import static com.google.common.base.Preconditions.checkNotNull;
 
 public abstract class BaseProfitBricksRequestBinder<T> implements MapBinder {
 
+   protected final String endpointPath = "/rest/v2";
```

Comment:
Thanks for your help, but I solved it in a different way, because I still kept getting further errors with the last injector snippet.
`protected final String endpoint = new ProfitBricksProviderMetadata().getEndpoint();`
and changed a little bit the request builder part
```R request = (R) fromRequest.toBuilder().endpoint(endpointPath).build();
      request = (R) request.toBuilder().replacePath(request.getEndpoint().getPath() + String.format("images/%s", imageId)).build();```